src/main/java/core/basesyntax/behaviour/Colorable.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.behaviour;
+
+import core.basesyntax.type.Color;
+
+public interface Colorable {
```

Comment:
Color getColor(); - is not used at all

String getStringColor(); - create interface and implement it just to map enum field to string is over-engineering

Please remove this interface